TODO

```diff
@@ -22,12 +22,12 @@ CONTENT todo
 
 1. integrate ksh into the shell chapters (assigned:paul)
 
-2. integrate dash into the shell chapters (obsolete no?)
+2. integrate dash into the shell chapters (obsolete no? its obsolete)
 
 3. extra Bash stuff
    - Array's (http://www.cyberciti.biz/faq/finding-bash-shell-array-length-elements/)
 
-4. add stuff from http://syntra.verwilst.be (we got permission) (assigned:serge)
+4. add stuff from http://syntra.verwilst.be (we got permission) (assigned:br0k3gl255)
```

Comment:
who/what's br0k3gl255?
I think you can scrap this point, that content has been away since along time.

Reply:
Br0k3ngl255 i my previous username, but it got hacked and I haven't used it since then.
I already have content about arrays and will incorporate it, later this week in content.

Reply:
OK, can you update that line?
No need to use GH usernames.

Reply:
done